disq/client.py

```diff
@@ -388,6 +388,13 @@ def addjob(self, queue, body, timeout_ms=0, replicate=0, delay_secs=0,
             args += [Token('ASYNC')]
 
         return self.execute_command(*args)
+        
+    def fastaddjob(self, queue, body, timeout_ms=0):
+        """
+        addjob without optional arguments.
+        """
+        args = ['ADDJOB', queue, body, timeout_ms]
+        return self.execute_command(*args)
```

Comment:
You can actually write this as `return self.execute_command('ADDJOB', queue, body, timeout_ms)` and skip allocating the `args` list.